content/en/blog/2024/otel-collector-survey/index.md

```diff
@@ -0,0 +1,93 @@
+- The vast majority deploy the Collector on Kubernetes (150/186)
+- There's more desire for stability (59), self-observability (53), and config management (59), than for new components (14)
+
+## Detailed Insights
+
+### Deployment Scale and Environment
+Our findings indicate robust use of the OTel Collector at scale, with 53.8%(100/186) of respondents deploying more than 10 collectors, and  22%(41/186) of respondents running between 2 and 5 collectors
+
+![Chart showing how many otel collectors people run in their organization](deployment-scale.png)
+
+Kubernetes is the leading platform for Collector deployment (80.6%), followed by virtual machines (33.3%) and bare metal (10.8%).
+
+![Chart showing where people deploy their otel collectors](deployment-environment.png)
+
+### Usage Scenarios 
+The OTel Collector is primarily used as a gateway (64.5%), demonstrating its pivotal role in aggregating telemetry data from various sources. Daemonsets (51.6%) and sidecars (23.7%) are also popular deployment models, showcasing the flexibility of the OTel Collector in different operational contexts.
+
+![Chart showing the usage scenarios of the otel collector](usage-scenarios.png)
+
+### Customization and Configuration
+A surprising number of people build their own distributions of the Collector (61/186), indicating that providing a composable Collector is important for the community. Most people who do build their own Collector distros use the [OTel Collector Builder (OCB)](https://github.com/open-telemetry/opentelemetry-collector-builder) (49/61). Of the 49 respondents who leverage the OCB, most were able to figure it out, with only 2 respondents stating that it’s too hard to use.
+
+![Chart showing how easy people find it to use the otel collector builder](ocb-usage.png)
+
+### Monitoring and Observability
+When it comes to monitoring the Collectors, a vast majority of respondents rely on Collector metrics and logs (70%), and only a few don't monitor their Collector(s) at all (16%). When we peered deeper into the data, we found that of the 125 respondents with more than 5 Collectors, only 15 are not monitoring their Collectors, and of the 100 respondents with more than 10 Collectors, only 9 are not monitoring their Collectors. This seems to indicate that users take monitoring their Collectors seriously once they reach a certain maturity with their Collector deployments.
+
+![Chart showing how people monitor their otel collector](monitoring.png)
+
+### OTel Components Usage
+The OTel Collector's flexibility is vividly illustrated by the wide array of exporters, receivers, processors, connectors, and extensions that are used across various environments. This highlights the Collector's capability to integrate with a vast range of tools and systems.
+
+The top components according to our survey results are as follows:
+
+#### Receivers
+- OTLP Receiver
+- Prometheus Receiver
+- File Receiver
+
+#### Exporters
```

Comment:
Why do we have 3 items in receivers, but only 2 in exporters, and then 5 on processors, 4 in connectors, and so on? Can we standardize on 5 for each component type? Here's a list of components, in order. Note that some of my data conflicts with what was shown here originally, so it might be good to have someone else to double-check the numbers. I would also use the full component name, to avoid ambiguity (Prometheus Exporter vs. Prometheus Remote Write Exporter, for instance). 

Exporter:
1. otlpexporter
2. prometheusremotewriteexporter
3. prometheusexporter
4. lokiexporter
5. debugexporter

Receivers:
1. otlpreceiver
2. prometheusreceiver
3. filelogreceiver
4. hostmetricsreceiver
5. k8sclusterreceiver

Processors:
1. batchprocessor
2. attributesprocessor
3. filterprocessor
4. memorylimiterprocessor
5. k8sattributesprocessor

Connectors:
1. spanmetricsconnector
2. servicegraphconnector
3. routingconnector
4. countconnector
5. datadogconnector

Extensions
1. healthcheckextension
2. basicauthextension
3. pprofextension
4. bearertokenauthextension
5. oauth2clientauthextension

Reply:
@cyrille-leclerc , I used your summaries for this list.

Reply:
Thanks @cyrille-leclerc for providing the list

Reply:
Updated the list

Reply:
Is it worth linking each of the components in this list to its readme?

Reply:
is this question still open?

Reply:
I ll wor on linking each of them to the readme